src/core/utils/file.ts

```diff
@@ -1,7 +1,11 @@
-import { promises as fs } from "fs";
+import dotenv from "dotenv";
+import { existsSync, promises as fs } from "fs";
+import os from "os";
 import path from "path";
 import { logger } from "./logger";
+import { ENV_FILENAME } from "./../constants";
 import { stripJsonComments } from "./strings";
+const { readFile } = fs;
 
```

Comment:
nit: move readFile to the following import

``import { existsSync, promises as fs } from "fs";``


``import { existsSync, readFile, promises as fs } from "fs";``

Reply:
we cant do this as we get readfile from {promises} not from "fs";